spec/controllers/idv/confirmations_controller_spec.rb

```diff
@@ -245,13 +245,16 @@ def index
       expect(@analytics).to have_logged_event(Analytics::IDV_DOWNLOAD_PERSONAL_KEY, success: true)
     end
 
-    it 'recovers pii with the code' do
+    it 'recovers pii and verifies personal key digest with the code' do
       get :show
       get :download
 
       code = response.body.chomp
 
-      expect(user.profiles.first.recover_pii(normalize_personal_key(code))).to be
```

Comment:
if we're no longer using `normalize_personal_key` we can remove `include PersonalKeyValidator` up top, and probably want to update the  spec on like 111 to check pii as well

Reply:
we are still using `normalize_personal_key`, but I added 0709cbb2f to have a similar check in that spec